iphone/cli/commands/_buildModule.js

```diff
@@ -237,6 +252,60 @@ iOSModuleBuilder.prototype.processLicense = function processLicense() {
 };
 
 iOSModuleBuilder.prototype.processTiXcconfig = function processTiXcconfig(next) {
+	const srcFile = path.join(this.projectDir, this.moduleName + '.xcodeproj', 'project.pbxproj');
+	const xcodeProject = xcode.project(path.join(this.projectDir, this.moduleName + '.xcodeproj', 'project.pbxproj'));
+	const contents = fs.readFileSync(srcFile).toString();
+	const appName = this.moduleIdAsIdentifier;
+
+	if (this.isFramework && !contents.includes('TitaniumKit.xcframework')) {
+		this.logger.warn(`Module created with sdk < 9.2.0, need to add TitaniumKit.xcframework. The ${this.moduleName}.xcodeproj has been updated.`);
+
+		xcodeProject.hash = xcodeParser.parse(contents);
+		const xobjs = xcodeProject.hash.project.objects,
```

Comment:
Just a style thing in JS - in the old days you should use `var` and declare all variables at the top of the scope, because JS hoisted them to the top of the scope anyways, so we wanted to match in source to avoid inconsistency. But with `let`/`const`, they're more like variables in any other language, they're more or less treated as declared/initialized where you define them, so you don't need to stick them at the top of the block/scope. Ideally we'd break up this big block of declared variables and move them to be declared where they're needed to make it easier to follow the logic and which grouping of variables applies to what lines of code used to edit/process them.

This is a long way of saying, most of these variables aren't needed until lines 294/299 below.